### PR TITLE
#1193 - NOTES - Closed Programs - Redirect every run to Eligibility Summary

### PR DESCRIPTION
**Pull request by C-Love**
There will be no more access to NOTES - Closed Programs

### CODE REVIEW

File: notes/closed-programs.vbs [hunk 1]
```diff
@@ -51,6 +51,7 @@ changelog = array()
 
 'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
 'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("03/13/2023", "* * * Closed Programs has been retired. * * *##~####~##NOTES - Eligibility Summary will now run every time you press the button for Closed Programs.##~##", "Casey Love, Hennepin County")
```
Comment (IlseFerris) [inline notes/closed-programs.vbs:54]: Do you want to update the date of the changelog?

Comment (C-Love) [reply notes/closed-programs.vbs:54]: I would love to.

Comment (C-Love) [reply notes/closed-programs.vbs]: @IlseFerris - ok - it is actually ready now.

**Review by IlseFerris — APPROVED** [review]
I approve thee!

**Pull request merged**